app/views/token-based-resume/select_multiple_applications.erb

```diff
@@ -17,26 +17,30 @@
                 <% @applications.each do |application| %>
                 <tr class="govuk-table__row">
                     <td class="govuk-table__cell govuk-table__cell--numeric">
-                        <% if application.is_submitted? %>
+                        <% if application.is_submitted? || application.is_cancelled? %>
                             <%= application.reference %>
                         <% else %>
                             <a href="/sponsor-a-child/resume-application/select?ref=<%= application.reference %>"><%= application.reference %></a>
                         <% end %>
                     </td>
-                    <td class="govuk-table__cell govuk-table__cell--numeric"><%= application.updated_at %></td>
+                    <td class="govuk-table__cell govuk-table__cell--numeric"><%= application.updated_at.in_time_zone("London").strftime("%d-%m-%Y at %I:%M%p") %></td>
                     <td class="govuk-table__cell govuk-table__cell--numeric">
                         <% if application.is_submitted? %>
                             Sent
                         <% else %>
-                            In progress
+                            <% if application.is_cancelled? %>
```

Comment:
I 🤔 we have to deal with in progress, submitted and cancelled?

Reply:
?

```
                        <% if application.is_submitted? %>
                            Sent
                        <% else %>
                            <% if application.is_cancelled? %>
                                Cancelled
                            <% else %>
                                In progress
                            <% end %>
                        <% end %>
```

this is what the code looks like now, not sure what you're looking at?

Reply:
🤣 Fair enough - apologies